nodl_python/nodl/types.py

```diff
@@ -10,10 +10,20 @@
 # You should have received a copy of the GNU Limited General Public License along with
 # this program. If not, see <http://www.gnu.org/licenses/>.
 
+from enum import Enum
 from typing import Any, Dict, List, Optional
 
-from nodl._util import qos_to_dict
-from rclpy.qos import QoSPresetProfiles, QoSProfile
+
+class PubSubDirection(Enum):
```

Comment:
I know it's a small enum, but I suggest using [unique](https://docs.python.org/3/library/enum.html#enum.unique) as a best-practice.

Reply:
In e.g. https://github.com/ubuntu-robotics/nodl/blob/37175605af43a1626ba09cefc1d891c590197305/nodl_python/nodl/_parsing/_v1/_parsing.py#L34 the value is used to pick an enum value based on the string in the xml attribute.

Reply:
Made unique in 42a43ab